common/eth2/src/lighthouse_vc/http_client.rs

```diff
@@ -78,14 +67,11 @@ pub fn parse_pubkey(secret: &str) -> Result<Option<PublicKey>, Error> {
 }
 
 impl ValidatorClientHttpClient {
-    /// Create a new client pre-initialised with an API token.
-    pub fn new(server: SensitiveUrl, secret: String) -> Result<Self, Error> {
+    /// Create a new client.
+    pub fn new(server: SensitiveUrl) -> Result<Self, Error> {
         Ok(Self {
             client: reqwest::Client::new(),
             server,
-            server_pubkey: parse_pubkey(&secret)?,
-            secret: Some(secret.into()),
-            authorization_header: AuthorizationHeader::Bearer,
```

Comment:
Ooh, this is deleting a little too much. We need to keep the `Authorization` header on both the client and server side. We just need to change the authentication token to an arbitrary byte string, rather than a token derived from the server's signing key.